### PR TITLE
Clarify that `pg_prove` needs to be installed

### PR DESCRIPTION
**Pull request by abitrolly**

### CODE REVIEW

File: doc/pgtap.mmd [hunk 1]
```diff
@@ -122,9 +122,8 @@ schema, use the `SCHEMA` clause to specify the schema, like so:
 Testing pgTAP with pgTAP
 ------------------------
 
-In addition to the PostgreSQL-standard `installcheck` target, the `test`
-target uses the `pg_prove` Perl program to do its testing, which will be
-installed with the
+`test` target uses the `pg_prove` Perl program to do its testing, which needs
```
Comment (theory) [inline doc/pgtap.mmd:125]: I think you deleted more words here than you mean to.

Comment (abitrolly) [reply doc/pgtap.mmd:125]: Why? From the code `test` target doesn't use `installcheck`. I thought it is a leftover.

Comment (theory) [reply doc/pgtap.mmd:125]: No, I mean you deleted the beginning of the sentence.

Comment (abitrolly) [reply doc/pgtap.mmd:125]: Maybe there is a misunderstanding. What's the point in telling that `test` uses `installcheck` if it doesn't use it?

Comment (theory) [reply doc/pgtap.mmd:125]: You deleted the words

> In addition to the PostgreSQL-standard `installcheck` target, the 

And I don't think that's what you meant to do.

Comment (abitrolly) [reply doc/pgtap.mmd]: Brought it back. Still makes the text sound wrong for me.

Comment (theory) [reply doc/pgtap.mmd]: LGTM.

**Pull request merged**